kolibri/utils/server.py

```diff
@@ -302,3 +305,32 @@ def get_urls(listen_port=None):
         return STATUS_RUNNING, urls
     except NotRunning as e:
         return e.status_code, []
+
+
+def show_url_info(port, daemon=True):
```

Comment:
Because this function is called `show_url_info` and handles the daemon logging stuff: Perhaps better to move the daemon logging logic back to where `become_daemon()` sits.